server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -1116,7 +1116,8 @@ protected Node(
                 fileCache,
                 taskCancellationMonitoringService,
                 resourceUsageCollectorService,
-                repositoryService
+                repositoryService,
+                analysisModule.getAnalysisRegistry()
```

Comment:
No need, please use `indicesService.getAnalysis()` in the constructor

Reply:
Good catch!